confdb/aclocal_libs.m4

```diff
@@ -108,7 +108,7 @@ dnl PAC_CHECK_HEADER_LIB_OPTIONAL(with_option, header.h, libname, function)
 dnl Check optional library. The results are in $pac_have_$1.
 AC_DEFUN([PAC_CHECK_HEADER_LIB_OPTIONAL],[
     PAC_SET_HEADER_LIB_PATH($1)
-    if test "${with_$1}" = "embedded" ; then
+    if test "${with_$1}" = "embedded" || test "${with_$1}" = "install"; then
```

Comment:
I'm not a shell language lawyer, but maybe the following is more portable alternative?

```suggestion
    if test "${with_$1}" = "embedded" -o "${with_$1}" = "install"; then
```

Reply:
I only learned recently, that it is the opposite - the `||` form is recommended and `-o` has been marked as obsolete - https://pubs.opengroup.org/onlinepubs/9699919799/utilities/test.html

![image](https://github.com/user-attachments/assets/8b99ed01-6ab6-45f6-9d0e-d2e90b66be34)